### PR TITLE
fixes initial zoom issues after loading the graph

### PR DESCRIPTION
**Pull request by witekw**
I've simplified the code and the probem is gone. Need react guru to look at this.

### CODE REVIEW

File: ui/client/components/graph/Graph.js [hunk 1]
```diff
@@ -40,16 +41,16 @@ class Graph extends React.Component {
     })
     this.nodesMoving();
 
-    this.espGraphRef = React.createRef()
-    this.parent = document.getElementById(this.props.parent)
+    this.myRef = React.createRef()
```
Comment (lciolecki) [inline ui/client/components/graph/Graph.js]: MyRef? Mayby better will be graphRef?

File: ui/client/containers/Visualization.js [hunk 2]
```diff
@@ -19,12 +19,25 @@ import RouteLeavingGuard from "../components/RouteLeavingGuard";
...
-    super(props);
-    this.state = {timeoutId: null, intervalId: null, status: {}, dataResolved: false};
-    this.graphRef = React.createRef();
-    this.bindShortCuts();
+    super(props)
+    this.state = {timeoutId: null, intervalId: null, status: {}, dataResolved: false}
+    this.bindShortCuts()
+
+    this.graphRef = React.createRef()
+    // this.pageRef = React.createRef()
```
Comment (lciolecki) [inline ui/client/containers/Visualization.js]: I think if don't use this code then we can remove it.

File: ui/client/components/graph/Graph.js [hunk 3]
```diff
@@ -26,7 +26,8 @@ class Graph extends React.Component {
     processToDisplay: PropTypes.object.isRequired,
     groupingState: PropTypes.array,
     loggedUser: PropTypes.object.isRequired,
-    connectDropTarget: PropTypes.func
+    connectDropTarget: PropTypes.func,
+    //pageRef: PropTypes.object.isRequired
```
Comment (lciolecki) [inline ui/client/components/graph/Graph.js]: We can remove it.

File: ui/client/components/graph/Graph.js (hunk 1 above)
Comment (witekw) [reply ui/client/components/graph/Graph.js]: Sure, its work in progress, I'll clean it up.

File: ui/client/components/graph/Graph.js [hunk 4]
```diff
@@ -593,20 +588,21 @@ function mapState(state, props) {
...
   return {
     divId: "esp-graph-subprocess",
-    parent: "modal-content",
     padding: 40,
     readonly: true,
     singleClickNodeDetailsEnabled: false,
     nodeIdPrefixForSubprocessTests: state.graphReducer.nodeToDisplay.id + "-", //TODO where should it be?
     processToDisplay: props.processToDisplay,
     processCounts: props.processCounts,
+    //pageRef: props.pageRef,
```
Comment (lciolecki) [inline ui/client/components/graph/Graph.js]: Remove not use code.

**Pull request merged**